sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -1713,8 +1721,12 @@ void CAFMaker::produce(art::Event& evt) noexcept {
         rec.reco.shw.push_back(SRShower());
         FillShowerVars(*thisShower[0], vertex, fmShowerHit.at(iPart), lar::providerFrom<geo::Geometry>(), producer, rec.reco.shw.back());
 
+        art::Ptr<anab::T0> thisPFPT0;
+        if (f1PFPT0.isValid()) {
+          thisPFPT0 = f1PFPT0.at(iPart);
+        }
```

Comment:
I defer to others here because I'm definitely not proclaiming to be an expert in efficiency, but do we want to initialize this variable similarly to the pfpMeta below? (I.e. const and with a ternary.) Maybe the answer is no. Does @PetrilloAtWork have wisdom here?